webapp-django/crashstats/crashstats/views.py

```diff
@@ -723,10 +723,6 @@ def crashes_per_day(request, default_context=None):
     for version in context['active_versions'][params['product']]:
         context['available_versions'].append(version['version'])
 
-    if not params.get('platforms'):
-        params['platforms'] = [
-            x['name'] for x in platforms if x.get('display')
-        ]
```

Comment:
Looks like this used to check to see if the user was specifying platforms and if not, it'd reduce the search to all displayed platforms. (I'm not clear on what "display" means in this context, but guessing it's what's in a drop-down somewhere.)